contentcuration/contentcuration/frontend/administration/pages/Users/UserActionsDropdown.vue

```diff
@@ -1,25 +1,29 @@
 <template>
 
   <div>
-    <ConfirmationDialog
-      v-model="deleteDialog"
-      title="Delete user"
-      :text="`Are you sure you want to permanently delete ${user.name}'s account?`"
-      confirmButtonText="Delete"
+    <KModal
+      v-if="deleteDialog"
+      :title="$tr('deleteUserTitle')"
+      :submitText="$tr('deleteAction')"
+      :cancelText="$tr('cancelAction')"
       data-test="confirm-delete"
-      @confirm="deleteHandler"
-    />
-    <ConfirmationDialog
-      v-model="deactivateDialog"
-      title="Deactivate user"
-      :text="
-        `Deactivating ${user.name}'s account will block them from ` +
-          `accessing their account. Are you sure you want to continue?`
-      "
-      confirmButtonText="Deactivate"
+      @submit="deleteHandler"
+      @cancel="deleteDialog = false"
+    >
+      <p>{{ $tr('deleteUserMessage', { name: user.name }) }}</p>
```

Comment:
In the administration pages, we don't use i18n strings because these pages will be visible only to the LE team. Could you please use unwrapped strings instead? (I know the issue mentions that all strings should be internationalized, but there is an exception with admin pages 😅)